### PR TITLE
Admin panel implementation

### PR DESCRIPTION
**Pull request by Somiel**
Backend admin panel implementation

### CODE REVIEW

File: app/admin/dashboard.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  content title: proc{ I18n.t("active_admin.dashboard") } do
+    div class: "blank_slate_container", id: "dashboard_default_message" do
+      span class: "blank_slate" do
+        span I18n.t("active_admin.dashboard_welcome.welcome")
+        small I18n.t("active_admin.dashboard_welcome.call_to_action")
+      end
+    end
+
+    # Here is an example of a simple dashboard with columns and panels.
```
Comment (murderface13) [inline app/admin/dashboard.rb]: remove commented code, pls

File: app/admin/user.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    column :first_name
+    column :last_name
+    column :phone_number
+    column :email
+    column :type
+    column 'Date', :updated_at
+    actions
+  end
+
+  controller do
```
Comment (murderface13) [inline app/admin/user.rb]: why do we need that controller actions?

File: app/admin/dashboard.rb (hunk 1 above)
Comment (Somiel) [inline app/admin/dashboard.rb]: done

File: app/admin/user.rb (hunk 2 above)
Comment (Somiel) [inline app/admin/user.rb]: just wanna make my controllers in AA do custom redirect after create/update actions

**Comment by murderface13** [comment]
:+1:

**Pull request merged**